psi4/driver/procrouting/dft_funcs/dict_dh_funcs.py

```diff
@@ -0,0 +1,442 @@
+#
+# Psi4: an open-source quantum chemistry software package
+#
+# Copyright (c) 2007-2017 The Psi4 Developers.
+#
+# The copyrights for code used from other parties are included in
+# the corresponding files.
+#
+# This file is part of Psi4.
+#
+# Psi4 is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, version 3.
+#
+# Psi4 is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License along
+# with Psi4; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# @END LICENSE
+#
+"""
+List of double-hybrid functionals
+"""
+
+funcs = []
+
+funcs.append({
+    "name": "B2PLYP",
+    "x_functionals": {
+        "GGA_X_B88": {
+            "alpha": 0.47
+        }
+    },
+    "x_hf": {
+        "alpha": 0.53
+    },
+    "c_functionals": {
+        "GGA_C_LYP": {
+            "alpha": 0.73
+        }
+    },
+    "c_mp2": {
+        "alpha": 0.27
+    },
+    "citation": '    S. Grimme, J. Chem. Phys., 124, 034108, 2006\n',
+    "description": '    B2PLYP Double Hybrid Exchange-Correlation Functional\n',
+})
+
+funcs.append({
+    "name": "DSD-BLYP",
+    "x_functionals": {
+        "GGA_X_B88": {
+            "alpha": 0.25
+        }
+    },
+    "x_hf": {
+        "alpha": 0.75
+    },
+    "c_functionals": {
+        "GGA_C_LYP": {
+            "alpha": 0.53
+        }
+    },
+    "c_mp2": {
+        "alpha": 1.0,
+        "os": 0.46,
+        "ss": 0.60
+    },
+    "citation": '    S. Kozuch, J.M.L. Martin, J. Comp. Chem., 34, 2327-2344, 2013\n',
+    "description": '    DSD-BLYP SCS Double Hybrid XC Functional\n',
+})
+
+funcs.append({
+    "name": "DSD-BLYP-D2",
+    "x_functionals": {
+        "GGA_X_B88": {
+            "alpha": 0.29
+        }
+    },
+    "x_hf": {
+        "alpha": 0.71
+    },
+    "c_functionals": {
+        "GGA_C_LYP": {
+            "alpha": 0.55
+        }
+    },
+    "c_mp2": {
+        "alpha": 1.0,
+        "os": 0.46,
+        "ss": 0.43
+    },
+    "citation": '    S. Kozuch, J.M.L. Martin, J. Comp. Chem., 34, 2327-2344, 2013\n',
+    "description": '    DSD-BLYP Dispersion-corrected SCS Double Hybrid XC Functional (-D2 parameters)\n',
+    "dispersion": {
+        "type": "d2",
+        "params": {
+            "s6": 0.35
+        },
+        "citation": "    S. Kozuch, J.M.L. Martin, J. Comp. Chem., 34, 2327-2344, 2013\n"
+    },
+})
+
+funcs.append({
+    "name": "CORE-DSD-BLYP",
+    "x_functionals": {"GGA_X_B88": {"alpha": 0.31}},
+    "x_hf": {"alpha": 0.69},
+    "c_functionals": {"GGA_C_LYP": {"alpha": 0.54}},
+    "c_mp2": {"alpha": 1.0, "os": 0.46, "ss": 0.37},
```

Comment:
We do, for normal DH functionals (like B2PLYP and PBE0-2). It is true that the presence of `"ss"` and `"os"` currently implies `"alpha" = 1`, but who knows what turns up later.

Reply:
Hmm, ok this is a bit awkward in how this works. If we are a `scs_hybrid` we use the `c_os/c_ss alpha`, otherwise we just add in the `MP2 correlation energy * alpha`. Currently we determine scs_hybrid by the following:

```
    bool is_c_scs_hybrid() const { return c_os_alpha_ != 0.0 || c_ss_alpha_ != 0.0; }
```

So the following will give the same result for any value of alpha.
```
"c_mp2": {"alpha": 0.5, "os": 0.46, "ss": 0.37}
```

What is your personal view of how this should work regardless of how the code currently is (we can fix that later)?

Reply:
I'm not knowledgeable enough about XC functionals to judge that. In my opinion, it doesn't hurt to have `alpha` explicit, but then it shouldn't get overridden in the code... Perhaps @susilehtola has an opinion?

Reply:
From above, lets just remove alpha for now from these dicts.